Sources/BraintreeCore/UIApplication+URLOpener.swift

```diff
@@ -33,4 +33,11 @@ extension UIApplication: URLOpener {
         }
         return canOpenURL(payPalURL)
     }
+
+    /// :nodoc: This method is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    /// Indicates whether the PayPal App is installed.
+    @_documentation(visibility: private)
+    public func openURL(_ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any], completionHandler completion: ((Bool) -> Void)?) {
```

Comment:
We always send an empty dictionary `[:]` for options. We can remove this property. I think what confused me was that this method signature is the same as Apple's but technically it's just our wrapper.

Reply:
Good call, updated: https://github.com/braintree/braintree_ios/pull/1360/commits/4c7043f0522f282e12da4fdf316fc012282f8e3b